documentation/book/assembly-kafka-connect-configuration.adoc

```diff
@@ -0,0 +1,28 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-connect.adoc
+// assembly-deployment-configuration-kafka-connect-s2i.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-connect-configuration-{context}']
+
+= Kafka Connect configuration
+
+{ProductName} allows you to customize the configuration of Apache Kafka Connect nodes.
```

Comment:
{ProductName} allows you to customize the configuration of Apache Kafka Connect nodes.
You can specify and configure most of the options listed in {ApachaKafkaConnectConfig}.

CHANGE TO

{ProductName} allows you to customize the configuration of Apache Kafka Connect nodes by editing most of the options listed in {ApachaKafkaConnectConfig}.